src/tags.ts

```diff
@@ -51,6 +51,21 @@ export interface TagsBase {
    */
   classification: SecurityClassification;
 
+  /**
+   * Criticality of the resources
+   */
+  criticality: 'critical' | 'high' | 'medium' | 'low';
+
+  /**
+   * THe responder team listed in OpsGenie.
+   * @see https://toitutewhenua.app.opsgenie.com/teams/list
+   */
+  responderTeam?: string;
+  /**
+   * collection git info, commit, version etc
+   */
```

Comment:
could we doc the default `@default false` ?

Reply:
good spot, updated.